spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -185,14 +177,11 @@ public ModelAndView home(Model model, HttpServletRequest request) throws IOExcep
         ModelAndView mav = new ModelAndView();
 
         if (auth.isAuthenticated()) {
-            User user = userRepository.findByUsername(auth.getName());
+            User user = userRepository.findByUsername(oAuthUserDetailsService.getLoginNameFromLoggedInUser());
 
             if (user != null) {
 
-                boolean end = false;
-                boolean found = false;
-
-                List<ServiceApi> services = serviceApiRepository.findByOwnerUsername(auth.getName());
+                List<ServiceApi> services = serviceApiRepository.findByOwnerUsername(oAuthUserDetailsService.getLoginNameFromLoggedInUser());
                 if (services != null && !services.isEmpty()) {
```

Comment:
This is the third nested if-statement. Rule of thumb a maximum nesting level of two to keep things readable. Could you extract a method to simplify this logic?

Reply:
TODO on PR #6